engine/source/editor/source/editor_ui.cpp

```diff
@@ -1020,7 +1041,7 @@ namespace Pilot
         if (selected_object == nullptr)
             return;
 
-        constexpr float angularVelocity = 18.0f / 600.0f;
+        float angularVelocity = 18.0f / Math::max(m_engine_window_size.x, m_engine_window_size.y); // 18 degrees while moving full screen
```

Comment:
I can see that the core code of this PR is this one line, but the PR contains 6 commits. Because this branch is quite old.